### ISSUE TITLE
Seed of Corruption does not trigger on unit death

### ISSUE THREAD

**Issue by RoFlNagII** [Duskhaven/Bug-Reports#496]
I have noticed on multiple occasions that seed of corruption seems to not trigger on unit death.

Tested with different low level mobs.

**Comment by GoldWarr** [comment]
yeah i can attest to this , seed of corruption is not triggering even with M+ mobs in case of mobs dieing.
